source/fem-dem/cfd_dem_coupling.cc

```diff
@@ -1655,24 +1655,11 @@ CFDDEMSolver<dim>::solve()
   while (this->simulation_control->integrate())
     {
       this->simulation_control->print_progression(this->pcout);
-      bool refinement_step;
-      if (this->simulation_parameters.mesh_adaptation.refinement_at_frequency)
```

Comment:
Just curious to know why is those if conditions are removed from the solvers and fem-dem?

Reply:
It's because in the `update_boundary_conditions` functions, the only condition to respect was whether boundary conditions are time dependent. This is already very restrictive and not expensive to check, so to simplify the code I remove the `refinement_step` boolean here. It had no effect on any test. I feel like the previous condition was difficult to understand and didn't bring much.

Reply:
If it does the same I like it! It is indeed much clearer :)